packages/cli/src/util/dev/server.ts

```diff
@@ -1533,18 +1544,10 @@ export default class DevServer {
             );
           }
         }
-      } catch (err) {
+      } catch (err: unknown) {
         // `startDevServer()` threw an error. Most likely this means the dev
         // server process exited before sending the port information message
         // (missing dependency at runtime, for example).
-        if (err.code === 'ENOENT') {
-          err.message = `Command not found: ${chalk.cyan(
-            err.path,
-            ...err.spawnargs
-          )}\nPlease ensure that ${cmd(err.path)} is properly installed`;
-          err.link = 'https://vercel.link/command-not-found';
-        }
```

Comment:
Should this be removed or should we keep it?

Reply:
Maybe we should pretty print the error

Reply:
Reverted it actually, since printing the error causes the test to fail since we search for "Error!" in the logs. We can address this in another PR.